### PR TITLE
Nr/misc updates

### PR DESCRIPTION
**Pull request by nick-romano**
# Description

This PR updates the readmes in the map-components samples to more closely match the readmes in the coding components samples.

### CODE REVIEW

**Comment by nick-romano** [comment]
note- the angular templates are not documented well in either package (coding-components & map-components), if we'd like to fill out detail there, I'd argue it's a separate PR.

File: packages/map-components/templates/vite/README.md [hunk 1]
```diff
@@ -1,3 +1,43 @@
 # Map components Vite template
 
-📁 **[Click here to download this directory as a ZIP file](https://download-directory.github.io?url=https://github.com/Esri/arcgis-maps-sdk-javascript-samples-beta/tree/main/packages/map-components/templates/vite)** 📁
\ No newline at end of file
+📁 **[Click here to download this directory as a ZIP file](https://download-directory.github.io?url=https://github.com/Esri/arcgis-maps-sdk-javascript-samples-beta/tree/main/packages/map-components/templates/vite)** 📁
+
+this project showcases how to integrate the map components using vite.
```
Comment (omarkawach) [inline packages/map-components/templates/vite/README.md:5]: this --> This

File: packages/map-components/templates/webpack/README.md [hunk 2]
```diff
@@ -1,3 +1,84 @@
...
+📁 **[Click here to download this directory as a ZIP file](https://download-directory.github.io?url=https://github.com/Esri/arcgis-maps-sdk-javascript-samples-beta/tree/main/packages/map-components/templates/webpack)** 📁
+
+This repository showcases how to integrate the map components using webpack.
+
+## Project setup
+
+### Install dependencies
+
+```
+yarn install
```
Comment (omarkawach) [inline packages/map-components/templates/webpack/README.md:12]: Favor `npm` for users

**Review by omarkawach — APPROVED** [review]
Looks good! A few small comments

File: packages/map-components/templates/vite/README.md (hunk 1 above)
Comment (andygup) [inline packages/map-components/templates/vite/README.md:5]: vite => Vite

File: packages/map-components/templates/webpack/README.md [hunk 3]
```diff
@@ -1,3 +1,84 @@
...
+    ...
+  ]
+};
+```
+
+## Resources
+
+[Webpack's getting started guide](https://webpack.js.org/guides/getting-started/)
+
+[Webpack 5's `experiments` option](https://webpack.js.org/configuration/experiments/)
```
Comment (andygup) [inline packages/map-components/templates/webpack/README.md:74]: @nick-romano I think you said we weren't using this? If so, pls remove.

File: packages/map-components/templates/webpack/README.md [hunk 4]
```diff
@@ -1,3 +1,84 @@
...
+
+### Generate the production-ready compiled code
+
+```
+yarn run build
+```
+
+#### JS
+
+Imported the components using [Stencil's instructions for integrating components without a JavaScript framework](https://stenciljs.com/docs/javascript).
```
Comment (andygup) [inline packages/map-components/templates/webpack/README.md:29]:
```suggestion
Imported the components using [Stencil's pattern for integrating components without a JavaScript framework](https://stenciljs.com/docs/javascript).
```

**Review by andygup — APPROVED** [review]
Minor change requests...

File: packages/map-components/templates/webpack/README.md (hunk 3 above)
Comment (nick-romano) [reply packages/map-components/templates/webpack/README.md:74]: wasn't me who brought that up. Don't have context on this, so will reserve that change to another PR

**Pull request merged**